stores/amazon.py

```diff
@@ -16,10 +16,10 @@
 from selenium.webdriver.support.ui import WebDriverWait
 
 from utils.debugger import debug
+from utils.discord_presence import buy_update, start_presence, searching_update
```

Comment:
remove this since it should be in the try below so we dont crash on start if discord isnt installed/running

Reply:
Okay, I reverted the import.